src/_actions/__tests__/AppConfigActions-test.js

```diff
@@ -0,0 +1,14 @@
+import { UPDATE_APP_CONFIG } from '../../_constants/ActionTypes';
+import {updateAppConfig} from '../AppConfigActions';
+import expect from 'expect';
+
+describe('AppConfigActions',()=>{
+  it('should be equal to the expectedAction',()=>{
+      const expectedAction = {
+        type: UPDATE_APP_CONFIG,
+        field: 'balance',
+        value: 3,
+      };
+      expect(updateAppConfig('balance',3)).toEqual(expectedAction);
```

Comment:
@nuruddeensalihu these are actually action creators.
Notice they are actually functions that return the action object.
I'd say the no logic ones will not fail.
There are others that can fail, they have api calls, but on the other hand they are hard to test.